### DISCUSSION TITLE
oo/interface

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#158] [Announcements]
# oo/interface

Java程序员进阶之路，小白的零基础Java教程，认真聊聊Java接口

https://tobebetterjavaer.com/oo/interface.html

**Comment by ForeUP** [comment]
在 定义接口 1）**接口不允许直接实例化** ，否则编译器会报错。 小节中给出了如下的代码：
```java
public class Computer implements Electronic {

    public static void main(String[] args) {
        new Computer();
    }

    @Override
    public int getElectricityUse() {
        return 0;
    }
}
```
其中的 `public static void main(String[] args)` 方法在这里好像用不到，反而让人觉得很奇怪，或者是我的理解有问题？

**Reply by wshwang7758** [reply]
老哥，这是main方法，Java程序的入口。啥叫用不到......
